src/commands/junit/upload.ts

```diff
@@ -325,6 +353,8 @@ UploadJUnitXMLCommand.addOption('env', Command.String('--env'))
 UploadJUnitXMLCommand.addOption('dryRun', Command.Boolean('--dry-run'))
 UploadJUnitXMLCommand.addOption('tags', Command.Array('--tags'))
 UploadJUnitXMLCommand.addOption('metrics', Command.Array('--metrics'))
+UploadJUnitXMLCommand.addOption('reportTags', Command.Array('--report-tags'))
+UploadJUnitXMLCommand.addOption('reportMetrics', Command.Array('--report-metrics'))
```

Comment:
Can you update the README too? https://github.com/DataDog/datadog-ci/blob/master/src/commands/junit/README.md#usage